src/index.css

```diff
@@ -59,7 +70,7 @@ p {
     float: right;
     height: 2vw;
     box-sizing: border-box;
-    background-color: white;
+    background-color: rgb(188, 240, 200);

```

Comment:
this feels too bright, considering the rest of our palette are dim pastels. change it to rgb(190, 215, 190) instead